nextcord/utils.py

```diff
@@ -1129,9 +1129,54 @@ def format_dt(dt: datetime.datetime, /, style: Optional[TimestampStyle] = None)
     """
     if not isinstance(dt, datetime.datetime):  # pyright: ignore[reportUnnecessaryIsInstance]
         raise InvalidArgument("'dt' must be of type 'datetime.datetime'")
+    return format_ts(int(dt.timestamp()), style=style)
+
+
+def format_ts(ts: int, /, style: Optional[TimestampStyle] = None) -> str:
+    """A helper function to format an :class:`int` unix timestamp for presentation within Discord.
+
+    This allows for a locale-independent way of presenting data using Discord specific Markdown.
+
+    +-------------+----------------------------+-----------------+
+    |    Style    |       Example Output       |   Description   |
+    +=============+============================+=================+
+    | t           | 22:57                      | Short Time      |
+    +-------------+----------------------------+-----------------+
+    | T           | 22:57:58                   | Long Time       |
+    +-------------+----------------------------+-----------------+
+    | d           | 17/05/2016                 | Short Date      |
+    +-------------+----------------------------+-----------------+
+    | D           | 17 May 2016                | Long Date       |
+    +-------------+----------------------------+-----------------+
+    | f (default) | 17 May 2016 22:57          | Short Date Time |
+    +-------------+----------------------------+-----------------+
+    | F           | Tuesday, 17 May 2016 22:57 | Long Date Time  |
+    +-------------+----------------------------+-----------------+
+    | R           | 5 years ago                | Relative Time   |
+    +-------------+----------------------------+-----------------+
+
+    Note that the exact output depends on the user's locale setting in the client. The example output
+    presented is using the ``en-GB`` locale.
+
+    .. versionadded:: 2.6
+
+    Parameters
+    ----------
+    ts: :class:`int`
+        The unix timestamp to format.
+    style: :class:`str`
+        The style to format the datetime with.
+
+    Returns
+    -------
+    :class:`str`
+        The formatted string.
+    """
+    if not isinstance(ts, int):  # pyright: ignore[reportUnnecessaryIsInstance]
```

Comment:
While I do understand that `format_dt` initially did this, _is_ this actually needed? `ts` is already typed as an integer so is there really a need to check if it's an integer a second time?